packages/geolocator/tizen/src/geolocator_tizen_plugin.cc

```diff
@@ -44,6 +49,27 @@ class GeolocatorTizenPlugin : public flutter::Plugin {
   virtual ~GeolocatorTizenPlugin() {}
 
  private:
+  void SetupGeolocatorServiceUpdatesChannel(
+      flutter::BinaryMessenger *messenger) {
```

Comment:
Suddenly curious...
``flutter::BinaryMessenger *messenger``
``flutter::BinaryMessenger* messenger``
Which one are we? doesn't it matter?

Reply:
I think both are fine If you choose one consistently within a single file. and I always turn on auto formatting in vscode.
https://google.github.io/styleguide/cppguide.html#Pointer_and_Reference_Expressions

Reply:
The former is preferred by the Google style and the latter is used by the Chromium style. I don't know why. This repo follows the Google style so the former is correct.

Reply:
That's a tragedy :cry: